### ISSUE TITLE
Layout issue with bootstrap grid layout

### ISSUE THREAD

**Issue by JeroenDeDauw** [wmde/software.wikimedia.de#32]
Happens only for certain screen widths.

Employee grid:

![such a question mark](https://cloud.githubusercontent.com/assets/2711781/23795846/f5437c88-0597-11e7-84f4-74374eae6a92.PNG)

Also happens on the homepage:

![hplayoutissue](https://cloud.githubusercontent.com/assets/146040/23821594/6ce04100-0637-11e7-903d-0d66b7ca5192.png)

**Comment by JeroenDeDauw** [comment]
@jdittrich do you have any idea how to fix this?

**Comment by jsoftwareengineering** [comment]
I tried fixing this using solutions found on this [https://stackoverflow.com/questions/19695784/how-can-i-make-bootstrap-columns-all-the-same-height](Stack Overflow question).

Couldn't find a quick fix. The javascript solution that I pushed to the branch equalHeightColumns works on Chromium, but breaks Firefox. 

I think a more substantial rewrite of the layout than adding a css class is required, but I'm no web dev.
